packages/client/hmi-client/src/App.vue

```diff
@@ -10,81 +10,20 @@
 			<component class="page" ref="pageRef" :is="Component" :key="route.path" />
 		</router-view>
 	</main>
-	<footer class="footer">
-		<img src="@assets/svg/uncharted-logo-dark.svg" alt="logo" class="ml-2" />
-		<div class="footer-group">
-			<a target="_blank" rel="noopener noreferrer" @click="isAboutModalVisible = true">About</a>
-			<a target="_blank" rel="noopener noreferrer" :href="documentation">Documentation</a>
-			<a target="_blank" rel="noopener noreferrer" href="https://terarium.canny.io/report-an-issue">
-				Report an issue
-			</a>
-			<a
-				target="_blank"
-				rel="noopener noreferrer"
-				href="https://terarium.canny.io/request-a-feature"
-			>
-				Request a feature
-			</a>
-		</div>
-	</footer>
-	<tera-modal
-		v-if="isAboutModalVisible"
-		class="modal"
-		@modal-mask-clicked="isAboutModalVisible = false"
-		@modal-enter-press="isAboutModalVisible = false"
-	>
-		<template #header>
-			<h4>About Terarium</h4>
-		</template>
-		<template #default>
-			<div class="about-modal-content">
-				<div class="about-section">
-					<img
-						src="@/assets/svg/terarium-logo.svg"
-						alt="Terarium logo"
-						class="about-terarium-logo"
-					/>
-				</div>
-				<div class="about-section">
-					<p class="constrain-width">
-						Terarium is a comprehensive modeling and simulation platform designed to help
-						researchers and analysts find models in academic literature, parameterize and calibrate
-						them, run simulations to test a variety of scenarios, and analyze the results.
-					</p>
-				</div>
-				<div class="about-section">
-					<img
-						src="@/assets/svg/uncharted-logo-official.svg"
-						alt="Uncharted Software logo"
-						class="about-uncharted-logo"
-					/>
-					<p class="constrain-width">
-						Uncharted Software provides design, development and consulting services related to data
-						visualization and analysis software.
-					</p>
-				</div>
-			</div>
-		</template>
-		<template #footer>
-			<div class="modal-footer">
-				<p>&copy; Copyright Uncharted Software {{ new Date().getFullYear() }}</p>
-				<Button class="p-button" @click="isAboutModalVisible = false">Close</Button>
-			</div>
-		</template>
-	</tera-modal>
+	<tera-footer class="footer" />
```

Comment:
```suggestion
	<footer><tera-footer /></footer>
```
The fact that we place it in a `<footer>` in the `App.vue` doesn't mean that `<terar-footer>` should be aware of that. It should be an `<aside>` or `<section>` component.